source/includes/_commands.md

```diff
@@ -245,324 +282,384 @@ message CreateDomain {
-            "dest_account_id": "nikolai@test",
-            "asset_id": "coin#test",
-						"description": "Salary payment",
-            "amount": {
-                "int_part": 20,
-                "precision": 0
-            }
+            "command_type": "GrantPermission",
+            "account_id": "takemiya@soramitsu",
+            "permission_name": "CanAddAssetQuantity"
         }
     ], …
 }
 ```
 
 Field | Description | Constraint
 -------------- | -------------- | --------------
-Source account ID | ID of account to withdraw asset from | already existent, `[a-z]{1,9}\@[a-z]{1,9}`
-Destination account ID | ID of account to send asset at | already existent, `[a-z]{1,9}\@[a-z]{1,9}`
-Asset ID | ID of asset to use | already existent, `[a-z]{1,9}\#[a-z]{1,9}`
-Amount | amount of asset to transfer | 0 < amount < max_uint256
+Account ID | id of account whom rights are granted | already existent, account_name@domain
+Permission name | name of granted permission | permission is defined
 
 ### Validation
 
-1. Source and destination accounts have this asset in their AccountHasAsset relations.
-2. Precision is formed right.
-3. Source account has this amount of asset to transfer and is not zero.
-4. Source account can transfer money, and destination account can recieve money (their roles have these permissions).
+1. Account exists
+2. Transaction creator is permitted to grant this permission
 
-## Append role
+## Remove signatory
 
 ### Purpose
 
-Purpose of _append role_ is to promote an account to some created role in the system, where role is a set or permissions account has to perform an action (command or query).
+Purpose of _remove signatory_ is to remove public key, associated with an identity, from an account
 
 ### Structure
 
 ```protobuf
-message AppendRole {
-   string account_id = 1;
-   string role_name = 2;
+message RemoveSignatory {
+    string account_id = 1;
+    bytes public_key = 2;
 }
 ```
 ```json
 {
     "commands": [
         {
-            "command_type": "AppendRole",
+            "command_type": "RemoveSignatory",
             "account_id": "takemiya@test",
-            "role_name": "Administrator"
+            "public_key": string(64)
         }
     ], …
 }
 ```
 
 Field | Description | Constraint
 -------------- | -------------- | --------------
-Account ID | id or account to append role to | already existent `[a-z]{1,9}\@[a-z]{1,9}`
-Role name | name of already created role | already existent
-Amount | amount of asset to transfer | 0 < amount < max_uint256
+Account ID | ID of account to delete signatory from | already existent, account_name@domain
+Public key | Signatory to delete | ed25519 public key
 
 ### Validation
 
-1. Role should exist in the system.
-2. Account triggering command should have permissions to append role.
+When signatory is deleted, we should check if invariant of **size(signatories) >= quorum** holds.
+Signatory should be added previously to the account.
```

Comment:
"should have been previously added to the account" is more grammatically correct.